OpenRA.Mods.Common/Widgets/ButtonWidget.cs

```diff
@@ -39,6 +39,8 @@ public class ButtonWidget : Widget
 		public bool Shadow = ChromeMetrics.Get<bool>("ButtonTextShadow");
 		public Color ContrastColorDark = ChromeMetrics.Get<Color>("ButtonTextContrastColorDark");
 		public Color ContrastColorLight = ChromeMetrics.Get<Color>("ButtonTextContrastColorLight");
+		private string clickSound = ChromeMetrics.Get<string>("ClickSound");
```

Comment:
Please make these public so that individual buttons can choose to override them. This applies to the other widget types too.